Documentation-Requirements.md

```diff
@@ -33,6 +34,25 @@ If there are multiple notes, bullets should be used:
 >
 > - I am important point *n*.
 
+# Files and command names
+
+All filenames and command names should be rendered in a fixed-format font
+using backticks:
+
+> Run the `foo` command to make it work.
+
+> Modify the `bar` option in file `/etc/baz/baz.conf`.
+
+Any options that need to be specified to the command should also be rendered
+in the same manner:
```

Comment:
Lines 46-47 suggested rewrite to stay active: 

Render any options that need to be specified to the command in the same manner: